turbo.json

```diff
@@ -23,16 +24,10 @@
     "VERCEL",
     "VITE_CLERK_*",
     "EXPO_PUBLIC_CLERK_*",
-    "REACT_APP_CLERK_*",
-    "CI_REPO_OWNER",
-    "CI_REPO_NAME",
-    "CI_COMMIT_SHA",
```

Comment:
`CI_COMMIT_SHA`, `CI_BRANCH, etc. being in globalDependencies will force all tasks to be invalidated on each new PR.